PXWeb/Controllers/Api2/NavigationApi.cs

```diff
@@ -54,24 +54,24 @@ public virtual IActionResult GetNavigationById([FromRoute][Required]string id, [
         /// <response code="200">Success</response>
         /// <response code="429">Error respsone for 429</response>
         [HttpGet]
-        [Route("/v2/navigation")]
+        [Route("/v1/navigation")]
```

Comment:
The spec is incorrect on this. It should be a api/v2 I will fix a change on this.

Reply:
Added a PR for this https://github.com/statisticssweden/PxApiSpecs/pull/9

Reply:
I have updated the routes in the controllers now